src/routes/components/components.json

```
@@ -193,6 +193,16 @@
 		"addedOn": "2021-07-12T00:00:00Z",
 		"category": "SvelteKit Adapters"
 	},
+	{
+		"title": "svelte-adapter-bun",
+		"url": "https://github.com/gornostay25/svelte-adapter-bun",
+		"description": "SvelteKit adapter that generates a standalone Bun server.",
+		"npm": "svelte-adapter-bun",
+		"addedOn": "2022-07-10T00:00:00Z",
+		"category": "SvelteKit Adapters",
+		"stars": 0,
+		"tags": ["ssr","integrations"],
```

There is a `, ` at the end of the line, making the JSON invalid
```suggestion
		"tags": ["ssr","integrations"]
```